src/components/Proposal/Create/CreateProposalPage.tsx

```diff
@@ -61,6 +61,8 @@ class CreateProposalPage extends React.Component<IProps, IStateProps> {
   }
 
   public async componentDidMount() {
+    document.addEventListener("keydown", this.handleKeyPress, false);
```

Comment:
Would this be better in `ModalPopup`?  If we don't think it appropriate for _all_ popups (why not?)  then it can be optionally enabled.

Reply:
CreateProposalPage cant use ModalPopup (as currently designed) since it uses <ModalRoute>, not <Modal>

Reply:
Ah, I see this is already in `ModalPopup`

Reply:
Does this mean that none of the many popups spawned by CreateProposalPage are use the common code you created here? Or, to what extent do they share code with other popups?

Reply:
that is true, they are not using the shared code at all right now. one thing that could potentially be shared is some of the styling... there is certainly more work that could be done there